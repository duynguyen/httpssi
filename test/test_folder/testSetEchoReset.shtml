<div>
	<!--#set var="myTest" value="testValue" -->
	<!--#echo var="myTest" -->
	<!--#set var="myTest" value="newTestValue" -->
	<!--#echo var="myTest" -->
</div>