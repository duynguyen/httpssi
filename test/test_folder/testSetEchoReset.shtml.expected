<div>
	testValue
	testValue
	newTestValue
	newTestValue
</div>